.travis.yml

```diff
@@ -1,19 +1,15 @@
-# see http://about.travis-ci.org/docs/user/languages/php/ for more hints
 language: php
 
-# list any PHP version you want to test against
 php:
-  # using major version aliases
-
-  # aliased to 5.3.29
+  - 5.3.3
   - 5.3
-  # aliased to a recent 5.4.x version
   - 5.4
-  # aliased to a recent 5.5.x version
   - 5.5
-  # aliased to a recent 5.6.x version
   - 5.6
+  - hhvm
   
-before_script: composer install  
+sudo: false
+
+install: travis_retry composer install --no-interaction --prefer-source
```

Comment:
Tell composer to prefer cloning with git rather than using github's api to download dists to avoid using precious api requests and busting the api limit for other users using the box later when they might actually have a better reason for using the api than us.